src/components/NFTBook/CrossSellDialog.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <Dialog
+    :open="open"
+    panel-container-class="phone:w-full laptop:w-full max-w-[400px]"
+    panel-component="CardV2"
+    panel-class="overflow-y-scroll shadow-lg px-[24px]"
+    header-text=""
+    :is-disabled-backdrop-click="false"
+    :has-close-button="false"
+  >
+    <div class="flex flex-col items-center">
+      <Label
+        preset="h4"
+        class="text-dark-gray"
+        :text="$t('nft_book_cross_sell_title')"
+      />
+
+      <NFTBookCover class="mt-[24px] h-[200px]" :src="productImageUrl" />
+
+      <Label class="mt-[16px]" preset="h5" :text="productName" />
+      <Label class="mt-[4px]" preset="p6" :text="productPriceWithFormat" />
+
+      <p class="mt-[16px] text-[14px] line-clamp-2">{{ productDescription }}</p>
+
+      <footer
+        class="flex flex-col items-stretch justify-center w-full mt-[20px] gap-[12px]"
+      >
+        <ButtonV2
+          preset="secondary"
+          :text="$t('nft_book_cross_sell_action_accept')"
+          @click.prevent="handleAccept"
+        />
+        <ButtonV2
+          preset="plain"
+          :text="$t('nft_book_cross_sell_action_reject')"
+          @click.prevent="handleReject"
+        />
+      </footer>
+    </div>
+  </Dialog>
+</template>
+
+<script>
+import { mapActions } from 'vuex';
+
+import { NFT_BOOK_PLATFORM_LIKER_LAND } from '~/constant';
+
+import { logTrackerEvent, logPurchaseFlowEvent } from '~/util/EventLogger';
+
+import nftOrCollectionMixin from '~/mixins/nft-or-collection';
+
+export default {
+  name: 'NFTBookCrossSellDialog',
+  mixins: [nftOrCollectionMixin],
+  props: {
+    open: {
+      type: Boolean,
+      default: false,
+    },
+    classId: {
+      type: String,
+      default: undefined,
+    },
+    collectionId: {
+      type: String,
+      default: undefined,
+    },
+    platform: {
+      type: String,
+      default: undefined,
+    },
+  },
+  watch: {
+    open: {
+      immediate: true,
+      handler(open) {
+        if (open) {
+          this.lazyFetchProductInfo();
+          logTrackerEvent(
+            this,
+            'NFT',
+            'nft_book_cross_sell_dialog_show',
+            this.productId,
+            1
+          );
+        }
+      },
+    },
+  },
+  methods: {
+    ...mapActions(['addBookProductToShoppingCart', 'uiPromptSuccessAlert']),
+    handleAccept() {
+      logTrackerEvent(
+        this,
+        'NFT',
+        'nft_book_cross_sell_dialog_accept',
+        this.productId,
+        1
+      );
+
+      // NOTE: Custom price is not supported for now
+      if (this.isCollection) {
+        const hasStock = this.collection?.stock;
+        if (!hasStock || this.collectionPrice === 0) {
+          this.handleReject();
+          return;
+        }
+
+        this.addBookProductToShoppingCart({
```

Comment:
Not sure if add to cart action should be performed by this dialog or from parent component